web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
@ersinciftci In what way is using GlobalProperties an anti pattern? Might be good to send a PR and update the CONTRIBUTING doc, explaining this anti pattern & what a better solution is: https://github.com/cBioPortal/cbioportal/blob/master/CONTRIBUTING.md#devops.

@mandawilson @ersinciftci What's slightly odd to me now is that when using the default `EXAMPLE.properties` you get a different set up then when using no `EXAMPLE.properties`. I think the behavior should be the same. So maybe better to have the sessions service running by default?